### PR TITLE
ci: add sweep job that runs avery day at 3 AM

### PR DESCRIPTION
**Pull request by ivan-savciuc**

### CODE REVIEW

File: .github/workflows/sweep.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Sweep
+
+on:
+  schedule:
+    - cron: "0 1 * * *"
```
Comment (Serpentiel) [inline .github/workflows/sweep.yml:5]: what time zone does it uses?
me and @mhoffm-aiven agreed that it should do that at 3.00 UTC

**Review by Serpentiel — CHANGES_REQUESTED** [review]

File: .github/workflows/sweep.yml (hunk 1 above)
Comment (mhoffm-aiven) [reply .github/workflows/sweep.yml:5]: its UTC

Comment (ivan-savciuc) [reply .github/workflows/sweep.yml:5]: by default docker Ubuntu image uses UTC, so we are good

**Comment by ivan-savciuc** [comment]
Why 3 AM, not  1 AM or any other time? My idea is to clean up leaked infrastructure by the end of the day, and the earliest convenient time is 1 AM.

**Review by mhoffm-aiven — DISMISSED** [review]
lgtm

**Review by Serpentiel — APPROVED** [review]
lgtm

**Pull request merged**